java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/InitializationData.java

```diff
@@ -64,16 +64,6 @@ public InitializationData clone() {
      */
     public java.util.function.BiConsumer<Runnable, Connection> executor;
 
-    /**
```

Comment:
This was not used anywhere.

I believe compactIdResolver is a C++-specific feature used by the Python/Ruby C++ extensions.